### ISSUE TITLE
Notice mark - VHF - symbol correct?

### ISSUE THREAD

**Issue by gert-MORLION** [IEHG/Portrayal-Catalogue#21]
There are two symbols for the obligation to use a certain VHF channel:

NMKREG15:  obligation to enter in a radiotelephone link on the channel as indicated on the board
![image](https://github.com/IEHG/Portrayal-Catalogue/assets/85863216/61dd6a27-211c-409d-b4d5-9c9e7b5ec271)

NMKREG15A: radiotelophone with channel
![image](https://github.com/IEHG/Portrayal-Catalogue/assets/85863216/ed1c3546-ff38-4e28-a2e8-8ab9798b1567)

In the first symbol, there is no room to put the channel on the board (as indicated in the definition). 

What shall we do with this?

**Comment by BerndBirklhuber** [comment]
We have to use the symbols without number. in 2.5 and 2.6 the number can only be seen in the pick report. For S-401 we want to create a LUA script that allows to create a portrayal that includes the correct number within the symbol.

**Comment by DeniseLaDue-IEHG** [comment]
I agree; with Bernd regarding the inclusion of the VHF channel in the portrayal in S-401.
